### ISSUE TITLE
getCounterpartyAlias() returns null on payments

### ISSUE THREAD

**Issue by OGKevin** [bunq/sdk_java#49]
## Steps to reproduce:
1. Execute the following code:
```java
        Payment.list(
            apiContext,
            userId,
            monetaryAccountId
        ).getValue().get(0).getCounterpartyAlias()
```

## What should happen:
1. `MonetaryAccountReference` reference is returned.

## What happens:
1. `null` is returned 

## Logs
- Logs `no logs`

## Extra info:
- Tested on [0.12.3](https://github.com/bunq/sdk_java/releases/tag/0.12.3)

## Reference
 - https://together.bunq.com/topic/java-payment

**Comment by OGKevin** [comment]
Cloud not reproduce this with the following test:
```java
  @Test
  public void counterPartyAliasNotNullTest() {
    Pagination pagination = new Pagination();
    pagination.setCount(PAGE_SIZE);

    List<Payment> allPayment = Payment.list(
        apiContext,
        userId,
        monetaryAccountId,
        pagination.getUrlParamsCountOnly()
    ).getValue();

    for (Payment payment : allPayment) {
      Assert.assertNotNull(payment.getCounterpartyAlias());
      Assert.assertFalse(payment.getCounterpartyAlias().isAllFieldNull());
    }
  }
```

On [0.12.4](https://github.com/bunq/sdk_java/releases/tag/0.12.4)

**Comment by defl** [comment]
Hey @OGKevin, as you requested in the bunqtogether thread I've tested this with 0.12.4 but I still have the issue. However I see that there's more activity after this so I assume you guys still found it?

**Comment by OGKevin** [comment]
@defl Could you please provide a code snippet on how to reproduce this as I still cant reproduce this issue.

**Comment by defl** [comment]
```
for(List<Payment> pp: bunqClient.getPayments().values()) {
    for(Payment payment: pp) {
        System.out.println(payment.getCounterpartyAlias());
    }
}
```

**Comment by OGKevin** [comment]
@defl `System.out.println(payment.getCounterpartyAlias())` prints null due to an error in its adapter. However the field its self is not null! Please try the following snippet and you will see:
```java
      System.out.println(payment.getCounterpartyAlias().isAllFieldNull());
```

If it returns `true` then indeed all fields are null and then this is indeed a bug, if it returns `false` then everything is almost working as expected :P 

Ill reopen the issue as the adapter is retuning `null` when the `toString()` and `toJson()` methods are called regardless the field having non `null` values.

**Comment by defl** [comment]
Thanks for fixing!

**Comment by OGKevin** [comment]
@defl no problem! Will be included in the 0.12.5 release 👍
